dracut/30ignition/ignition-mount.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Ignition (mount)
+DefaultDependencies=false
+
+# Make sure /sysroot is mounted first, since we're mounting under there
+Requires=initrd-root-fs.target
+After=initrd-root-fs.target
+
+# Make sure root filesystem is remounted read-write if needed
+Requires=ignition-remount-sysroot.service
+After=ignition-remount-sysroot.service
+
+# Make sure we umount before switching root
+Conflicts=initrd-switch-root.target umount.target
+Conflicts=dracut-emergency.service emergency.service emergency.target
+
+# This ensures that we fail the boot if the umount stage fails
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=/run/ignition.env
+ExecStart=/usr/bin/ignition --root=/sysroot --platform=${PLATFORM_ID} --stage=mount --log-to-stdout
+ExecStop=/usr/bin/ignition --root=/sysroot --platform=${PLATFORM_ID} --stage=umount --log-to-stdout
```

Comment:
Thoughts on splitting this up into two units `mount` and `umount`?

Reply:
i think he had it that way and was asked to change it: see https://github.com/coreos/ignition-dracut/pull/47#discussion_r261385435

Reply:
Ah, yup.